src/style.css

```diff
@@ -1,3 +1,79 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+html {
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
 }
+
+img {
+  max-width: 100%;
+  display: block;
+}
+
+.header {
+  margin: 0 auto;
+  max-width: 1200px;
+  min-width: 1024px;
+  height: 60px;
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background: #fff;
```

Comment:
looks bad because of this
<img width="256" alt="Знімок екрана 2023-08-07 о 10 36 18" src="https://github.com/mate-academy/layout_moyo-header/assets/106310796/6f06efad-a27a-4210-966a-65fda2e9e739">